client/lib/generate-password/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+
+import { random, sample } from 'lodash';
+
+export const LETTER_CHARSETS = {
```

Comment:
Perhaps we can remove the `export` clause of all those consts that don't need to be consumed externally?

Reply:
fixed in https://github.com/Automattic/wp-calypso/pull/48762/commits/2c1fc92b1cbd60872d01e22380933f382b6d5e2d